src/server/scripts/Spells/spell_holiday.cpp

```diff
@@ -410,6 +411,84 @@ class spell_pilgrims_bounty_buff_food : public SpellScriptLoader
         }
 };
 
+enum FeastOnSpells
+{
+    FEAST_ON_TURKEY                     = 61784,
+    FEAST_ON_CRANBERRIES                = 61785,
+    FEAST_ON_SWEET_POTATOES             = 61786,
+    FEAST_ON_PIE                        = 61787,
+    FEAST_ON_STUFFING                   = 61788,
+    SPELL_CRANBERRY_HELPINS             = 61841,
+    SPELL_TURKEY_HELPINS                = 61842,
+    SPELL_STUFFING_HELPINS              = 61843,
+    SPELL_SWEET_POTATO_HELPINS          = 61844,
+    SPELL_PIE_HELPINS                   = 61845,
+    SPELL_ON_PLATE_EAT_VISUAL           = 61826
+};
+
+class spell_pilgrims_bounty_feast_on : public SpellScriptLoader
+{
+    public:
+        spell_pilgrims_bounty_feast_on() : SpellScriptLoader("spell_pilgrims_bounty_feast_on") { }
+
+        class spell_pilgrims_bounty_feast_on_SpellScript : public SpellScript
+        {
+            PrepareSpellScript(spell_pilgrims_bounty_feast_on_SpellScript);
+
+            void HandleDummy(SpellEffIndex /*effIndex*/)
+            {
+                Unit* caster = GetCaster();
+
+                uint32 _spellId = 0;
+                switch (GetSpellInfo()->Id)
+                {
+                    case FEAST_ON_TURKEY:
+                        _spellId = SPELL_TURKEY_HELPINS;
+                        break;
+                    case FEAST_ON_CRANBERRIES:
+                        _spellId = SPELL_CRANBERRY_HELPINS;
+                        break;
+                    case FEAST_ON_SWEET_POTATOES:
+                        _spellId = SPELL_SWEET_POTATO_HELPINS;
+                        break;
+                    case FEAST_ON_PIE:
+                        _spellId = SPELL_PIE_HELPINS;
+                        break;
+                    case FEAST_ON_STUFFING:
+                        _spellId = SPELL_STUFFING_HELPINS;
+                        break;
+                    default:
+                        return;
+                }
+
+                if (Vehicle* vehicle = caster->GetVehicleKit())
+                    if (Unit* target = vehicle->GetPassenger(0))
+                        if (Player* player = target->ToPlayer())
+                        {
+                            player->CastSpell(player, SPELL_ON_PLATE_EAT_VISUAL, true);
+                            caster->CastSpell(player, _spellId, true, NULL, NULL, player->GetGUID());
+                        }
+
+                if (Aura* aura = caster->GetAura(GetEffectValue()))
+                {
+                    if (aura->GetStackAmount() == 1)
+                        caster->RemoveAurasDueToSpell(aura->GetSpellInfo()->Effects[EFFECT_0].CalcValue());
```

Comment:
i have 2 auras to remove.
for instance, `caster->GetAura(GetEffectValue())` returns http://wotlk.openwow.com/spell=61804
if is its last charge, `aura->GetSpellInfo()->Effects[EFFECT_0].CalcValue()` returns http://wotlk.openwow.com/spell=61798
and I remove them both